universal-login-example/src/components/ApproveConnection.js

```diff
@@ -2,12 +2,17 @@ import ApproveConnectionView from '../views/ApproveConnectionView';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import {Wallet} from 'ethers';
+import DEFAULT_PAYMENT_OPTIONS from '../../config/defaultPaymentOptions';
 
 class ApproveConnection extends Component {
   constructor(props) {
     super(props);
     this.sdk = this.props.services.sdk;
     this.identityService = this.props.services.identityService;
+    this.state = {
+      backupCode: '',
+      btnLabel: 'RECOVER ACCOUNT'
```

Comment:
Where should it live? Could it be a link off of the requesting connection screen?

Reply:
It already exists on that screen. When you type an existing name it gives you two options: connect or recover

<img width="555" alt="screen shot 2018-10-04 at 12 05 44 pm" src="https://user-images.githubusercontent.com/112898/46483379-e3456880-c7cd-11e8-89e9-bcdf011e90ce.png">

But since some users may not find it, we should probably also add a link from the connection requested screen

Reply:
Okay, I'll hook up a link from the connection requested screen for now and then will try my hand at implementing this.